### PR TITLE
docs(hub) update prometheus docs to match version 0.9.0

### PR DESCRIPTION
**Pull request by gszr**
Update Prometheus plugin docs to match current version (0.9.0). Please review `index.md`, which contains the latest version of the docs.

### CODE REVIEW

**Comment by tlblessing** [comment]
@gszr shouldn't the PR title be Prometheus rather than AWS Lambda?

**Comment by gszr** [comment]
@tlblessing You are right; I will make sure to squash and merge once approved, to fix the commit message.

File: app/_hub/kong-inc/prometheus/0.8.x.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+1. Kong Enterprise users can protect the admin `/metrics` endpoint with an
+   [RBAC user](/enterprise/latest/setting-up-admin-api-rbac) that the
+   Prometheus servers use to access the metric data. Access through any
+   firewalls would also need to be configured.
+
+2. You can proxy the Admin API through Kong itself then use plugins to limit
+   access. For example, you can create a route `/metrics` endpoint and have
+   Prometheus access this endpoint to slurp in the metrics while preventing
+   others from access it. The specifics of how this is configured will depend
+   on your specific setup. Read the docs [Securing the Admin
```
Comment (tlblessing) [inline app/_hub/kong-inc/prometheus/0.8.x.md:183]:
```suggestion
   on your specific setup. For details, see [Securing the Admin
```

File: app/_hub/kong-inc/prometheus/0.8.x.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+   API](https://docs.konghq.com/latest/secure-admin-api/#kong-api-loopback) for
```
Comment (tlblessing) [inline app/_hub/kong-inc/prometheus/0.8.x.md:184]:
```suggestion
   API](https://docs.konghq.com/latest/secure-admin-api/#kong-api-loopback).
```

File: app/_hub/kong-inc/prometheus/0.8.x.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+   details.
```
Comment (tlblessing) [inline app/_hub/kong-inc/prometheus/0.8.x.md:185]:
```suggestion
 
```

File: app/_hub/kong-inc/prometheus/0.8.x.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+
+3. Finally, you could serve the content on a different port with a custom server
+   block using a [custom Nginx
+   template](/latest/configuration/#custom-nginx-configuration) with Kong.
+
+    The following block is an example custom nginx template which can be used
```
Comment (tlblessing) [inline app/_hub/kong-inc/prometheus/0.8.x.md:191]:
```suggestion
    The following block is an example custom nginx template that can be used
```

https://www.grammarly.com/blog/which-vs-that/

**Comment by gszr** [comment]
@tlblessing I've applied your suggestions on `index.md`, rather than `0.8.x.md` -- and also amended the commit message. Thanks!

File: app/_hub/kong-inc/prometheus/index.md [hunk 5]
```diff
@@ -35,18 +36,18 @@ params:
   protocols: ["http", "https", "tcp", "tls", "grpc", "grpcs"]
   dbless_compatible: yes
   dbless_explanation: |
-    The database will always be reported as “reachable” in prometheus with DB-less.
+    The database will always be reported as “reachable” in Prometheus with DB-less.
```
Comment (tlblessing) [inline app/_hub/kong-inc/prometheus/index.md:39]:
```suggestion
    The database will always be reported as reachable in Prometheus with DB-less.
```

don't need quotes; missed this the first go round

File: app/_hub/kong-inc/prometheus/0.8.x.md [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+  protocols: ["http", "https", "tcp", "tls", "grpc", "grpcs"]
+  dbless_compatible: yes
+  dbless_explanation: |
+    The database will always be reported as “reachable” in prometheus with DB-less.
+
+---
+
+Metrics are available on the Admin API at the `http://localhost:8001/metrics`
+endpoint. Note that the URL to the Admin API will be specific to your
+installation; see _Accessing the metrics_ below.
```
Comment (tlblessing) [inline app/_hub/kong-inc/prometheus/0.8.x.md:44]: @gszr Missed this one the first time as well. This should be a link:

```suggestion
installation. See  [Accessing the metrics](#accessing-the-metrics) .
```

**Comment by gszr** [comment]
> @gszr found a few little things I missed; suggested edits.

Fixed. Thanks!

> Question: So was the present 1.0.x version incorrect?
> image

Yes, the plugin never had a 1.0 release -- latest is 0.9.0, and the one mistakenly referred to as 1.0 is 0.8.

**Pull request merged**